File: UserStory/UserStory/Models/ModelTask/TaskCreationDTO.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using UserStory.Models.ModelFunctionality;

namespace UserStory.Models.ModelTask
{
    public class TaskCreationDTO
    {
        //public Guid TaskId { get; set; }
        public string TextTask { get; set; }
        public Guid FunctionalityId { get; set; }

        public Guid? SprintId { get; set; }
    }
}

File: FrontendTemplateProject-main/src/app/models/UserStoryModel/taskCreationDto.ts
import { Guid } from 'guid-typescript';

export class TaskCreationDto {
    
    textTask!: string;
    functionalityId!: Guid;

    constructor(
        textTask: string,
        functionalityId: Guid
    ) {
        this.textTask = textTask;
        this.functionalityId = functionalityId;
    }
    
}